packages/api-client/src/store/import-spec.ts

```diff
@@ -18,12 +17,6 @@ export function importSpecFileFactory({
   const importSpecFile = async (
     _spec: string | Record<string, any>,
     workspaceUid = 'default',
-    /**
-     * TODO: What do these look like?
-     * Ideally we reference some existing UIDs in the store and
-     * attach those as needed to entities below
-     */
-    overloadServers?: Spec['servers'],
   ) => {
```

Comment:
We wont need this overload servers anymore, we will just update the ones on the collection

Reply:
I think this is needed to overwrite what’s in the OpenAPI document through configuration. Same schema, but different place to add them.

Reply:
Going to rollback these changes and tackle it in a follow up PR